articles/connections/passwordless/android.md

```diff
@@ -1,6 +1,13 @@
 ---
 title: Using Passwordless Authentication on Android
 ---
+# Passwordless Authentication on Android
```

Comment:
@jeffreylees This has 2 H1s, can you fix that?

https://auth0-docs-content-pr-4455.herokuapp.com/docs/connections/passwordless/android